main.go

```diff
@@ -39,12 +40,19 @@ func main() {
 		log.Fatalf("failed to load AWS default config: %v", err)
 	}
 
-	app, err := app.New(ctx,
+	appConfigs := []app.ConfigOption{
 		app.WithExtensionName(filepath.Base(os.Args[0])),
 		app.WithLambdaRuntimeAPI(os.Getenv("AWS_LAMBDA_RUNTIME_API")),
 		app.WithLogLevel(os.Getenv("ELASTIC_APM_LOG_LEVEL")),
 		app.WithAWSConfig(cfg),
-	)
+	}
+
+	disableFnLog, _ := strconv.ParseBool(os.Getenv("ELASTIC_DISABLE_FUNCTION_LOG_SUBSCRIPTION"))
```

Comment:
Initially, I kept a negative flag to keep it `false` by default since not providing the config seemed more aligned with a `false` value. Agree with double-negs being confusing to users, I have updated it to use `ELASTIC_APM_LAMBDA_CAPTURE_LOGS`.